commands/Music/play.js

```diff
@@ -1,226 +1,145 @@
-const { Command } = require("klasa");
+const MusicCommand = require("../../lib/structures/MusicCommand");
 const { MessageEmbed } = require("discord.js");
 
-module.exports = class extends Command {
+module.exports = class extends MusicCommand {
 
     constructor(...args) {
         super(...args, {
+            requireDJ: true,
             runIn: ["text"],
```

Comment:
pointless

Reply:
No, because the inhibitor does not check this command as it doesn't have `requireMusic` for obvious reasons.

Reply:
no lol